model_compression_toolkit/core/common/mixed_precision/sensitivity_evaluation.py

```diff
@@ -236,6 +237,10 @@ def _compute_gradient_based_weights(self) -> np.ndarray:
                 assert len(compare_point_to_trace_hessian_approximations[target_node][image_idx]) == 1
                 # Append the single approximation value to the list for the current image
                 approx_by_image_per_interest_point.append(compare_point_to_trace_hessian_approximations[target_node][image_idx][0])
+
+            if self.quant_config.norm_weights:
+                approx_by_image_per_interest_point = hessian_utils.normalize_weights(approx_by_image_per_interest_point, self.output_nodes_indices)
```

Comment:
pass alpha with default value (although it has a default value in the function arguments, just for clarity)